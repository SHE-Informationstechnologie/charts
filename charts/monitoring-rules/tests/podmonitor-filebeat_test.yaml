suite: test podmonitor filebeat
templates:
  - podmonitor-filebeat.yaml
tests:
  - it: should work
    set:
     logging.enabled: true
     logging.podMonitor.enabled: true
    asserts:
      - isKind:
          of: PodMonitor
  - it: manifest should match snapshot
    set:
     logging.enabled: true
     logging.podMonitor.enabled: true
    asserts:
      - matchSnapshot: {}
