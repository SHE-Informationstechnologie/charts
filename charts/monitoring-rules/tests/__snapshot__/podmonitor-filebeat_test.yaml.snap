manifest should match snapshot:
  1: |
    apiVersion: monitoring.coreos.com/v1
    kind: PodMonitor
    metadata:
      labels:
        app.kubernetes.io/instance: RELEASE-NAME
        app.kubernetes.io/managed-by: Helm
        app.kubernetes.io/name: monitoring-rules
        app.kubernetes.io/version: 2.0.8
        helm.sh/chart: monitoring-rules-2.0.8
        release: kube-prometheus-stack
      name: RELEASE-NAME-monitoring-rules-logging-filebeat
      namespace: logging
    spec:
      podMetricsEndpoints:
        - path: /metrics
          port: metrics
      selector:
        matchLabels:
          k8s-app: filebeat
